src/components/search/SearchInterface.vue

```diff
@@ -71,10 +71,21 @@
     </b-row>
     <b-row
       v-if="hasAnyResults"
-      class="mb-3"
+      class="mb-3 "
+      :class="{ 'd-flex align-items-center': entityHeaderCardsEnabled }"
     >
       <b-col>
-        <p data-qa="total results">
+        <div
+          v-if="entityHeaderCardsEnabled"
```

Comment:
Let's change this condition to `v-if="contextLabel"`, and have the check for whether the feature is enabled go into the parent view instead, i.e. into the entity page.